changes/issue-8558-osquery-perf-os

```diff
@@ -0,0 +1 @@
+* Added support to simulate Windows 11 and Ubuntu 22.04 hosts (in addition to macOS) with the `osquery-perf` command, via the `--os_templates` flag.
```

Comment:
@mna Is the use of the `--os_templates` flag documented? If not, would you please update the `osquery-perf` readme to include usage instructions? Thanks!

Reply:
Ah yeah, good call, it is documented as part of `osquery-perf -h` output, but it's definitely worth a mention in the readme, I'll add this.